src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.cs

```diff
@@ -33,79 +24,47 @@ namespace Microsoft.CodeAnalysis
     /// this is a green node of Solution like ProjectState/DocumentState are for
     /// Project and Document.
     /// </summary>
-    internal partial class SolutionState
+    internal sealed partial class SolutionState
```

Comment:
this file is effectively all moves of compilation functionality moved to SOlutionCompilationState.  But there are a few things i've noted within.